src/android/io/sentry/SentryCordova.java

```diff
@@ -380,5 +387,30 @@ private SentryLevel getSentryLevelFromString(String level) {
     }
   }
 
+  private final String androidSdk = "sentry.java.android";
+  private final String nativeSdk = "sentry.native";
```

Comment:
I'd recommend either making static too as a constant or to use these strings directly in the switch statement as its not being reused